runtime/include/chpl-cache.h

```diff
@@ -141,6 +141,10 @@ void chpl_cache_invalidate(c_nodeid_t node, void* raddr, size_t size,
 }
 #endif
 
+#ifdef __cplusplus
+}
+#endif
```

Comment:
Looking at this PR with whitespace ignored shows this as just adding a new `}`.  That seems weird to me - if there was always a `}` missing, shouldn't it have failed before?  Won't it cause failures now?  Was this prompted by something you encountered?

Reply:
I suspect this change is not right and compiling with `--cache-remote` would show a problem. The `}` for `extern "C" {` is just above and this change is adding a duplicate one.

Reply:
Yes, good catch @lydia-duncan, this extra snippet got added when I was merging